lib/MyChartAPI.js

```diff
@@ -89,38 +89,74 @@ async function GetSchedule(
     });
 }
 
+/**
+ * Add relevant site info to the results object
+ * @param {object} results
+ * @param {object} defaultResults
+ * @param {string} siteKey
+ * @param {object} siteInfo
+ */
+function AddSiteInfo(results, defaultResults, siteKey, siteInfo) {
+    // set default availability values
+    results[siteKey] = JSON.parse(JSON.stringify(defaultResults));
+    if (Object.keys(siteInfo) == 0) return;
```

Comment:
are we checking for length here? IDK when Object.keys() will return 0.
Also, we should use `===`